src/protocols/secure_channel/CASESession.h

```diff
@@ -61,6 +61,19 @@ class DLL_EXPORT CASESession : public Messaging::UnsolicitedMessageHandler,
                                public PairingSession
 {
 public:
+    // Public so it is accessible to unit test
```

Comment:
Since public APIs are really meant to be consumed by others consumers in the SDK and/or applications, elevating what is truly a private function/enum so that a unit test can consume it feels like a dilution of the public construct.

I've generally espoused doing `friend`ing of the `Test*` class itself. This works out better as well, since the test classes usually need to access a lot more of the private state of the object to do more concerted white-box testing, or actuation of specific bits of logic private to the class (sparingly however).